packages/sdk/src/webHost.ts

```diff
@@ -57,10 +61,41 @@ export class WebHost {
     }
 
     private serveStaticFiles(server: Restify.Server) {
-        // Setup static files route
-        server.get('/*', Restify.plugins.serveStatic({
-            directory: this._baseDir,
-            default: 'index.html'
-        }));
+        server.get('/*',
+            // host static binaries
+            (req, res, next) => this.serveStaticBuffers(req, res, next),
+            // host static files
+            Restify.plugins.serveStatic({
+                directory: this._baseDir,
+                default: 'index.html'
+            }
+            ));
+    }
+
+    private readonly bufferRegex = new RegExp(`^/${BUFFER_KEYWORD}/(.+)$`);
+
+    private serveStaticBuffers(req: Restify.Request, res: Restify.Response, next: Restify.Next) {
+        // grab path part of URL
+        const matches = this.bufferRegex.exec(req.url);
+        const procPath = matches && matches[1] || null;
+
+        // see if there's a handler registered for it
+        if (!procPath || !this.bufferMap[procPath]) {
+            return next();
+        }
+
+        // if so, serve binary
+        res.sendRaw(200, this.bufferMap[procPath]);
```

Comment:
Oh missed this in the other PR, it is more correct to always call next(), even after res.send.